src/algorithms/spectral/mfcc.cpp

```diff
@@ -60,6 +60,11 @@ const char* MFCC::description = DOC("This algorithm computes the mel-frequency c
 "\n"
 "This algorithm depends on the algorithms MelBands and DCT and therefore inherits their parameter restrictions. An exception is thrown if any of these restrictions are not met. The input \"spectrum\" is passed to the MelBands algorithm and thus imposes MelBands' input requirements. Exceptions are inherited by MelBands as well as by DCT.\n"
 "\n"
+"IDCT can be used to compute smoothed Mel Bands. In order to do this: \n "
+"- compute MFCC\n"
+"- smoothedMelBands = 10^(IDCT(MFCC)/20) \n"
+"Note: The second step assumes that the default 'logType' were used to compute MFCCs, otherwise that formula should be changed in order to be consistent.\n"
```

Comment:
explicitly write the 'logType' value just in case we will change default value in future